langchain-core/src/messages/base.ts

```diff
@@ -219,6 +236,8 @@ export abstract class BaseMessage
    */
   id?: string;
 
+  tool_calls?: never[] | ToolCall[];
```

Comment:
```suggestion
  tool_calls?: ToolCall[];
```
I'm likely misunderstanding typescript `never[]` type but shouldn't this just be this, and it'll be undefined in most cases?

Reply:
No, I still want this to throw a type error:

```ts
if (isHumanMessage(msg)) {
  // This should error
  message.tool_calls?.[0].args
}
```

Reply:
why? shouldn't it always be undefined it it's a humanmessage?